test/integration/newapp_test.go

```diff
@@ -1711,7 +1712,7 @@ func TestNewAppSourceAuthRequired(t *testing.T) {
 	}
 }
 
-func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) string {
+func setupLocalGitRepo(t *testing.T, passwordProtected bool, requireProxy bool) (string, func()) {
 	// Create test directories
 	testDir, err := ioutil.TempDir("", "gitauth")
```

Comment:
>  Because just changing the location doesn't delete them

When we place them under `$BASETMPDIR`, we can take advantage of the [code](https://github.com/openshift/origin/blob/master/hack/lib/util/environment.sh#L149-L159) that runs around our suites and cleans up there. If you believe these files would _never_ be useful to debug this test if it flakes out, then I guess we could delete them in Go as well, however, again, we have chosen to always cleanup the previous test's stuff on the next test's run just to make sure we don't accidentally get rid of useful artifacts. I'll defer to your judgment there but I will continue to prefer the cleanup-on-startup paradigm.